src/app/round-template.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {RoundTemplate, roundTemplates} from './round-template';
+
+describe('Round Templates', () => {
+  roundTemplates.forEach(({name, flowerSpecies, startTime, endTime, tickSpeed, bees}) => {
+    describe(name, () => {
+
+      it('has 16 flowers', () => {
+        expect(flowerSpecies.length).toBe(16);
+      });
+
+      if (bees) {
+        describe('if there are bees', () => {
+          it('has at least 1 bee', () => {
+            expect(bees.length).toBeGreaterThanOrEqual(1);
+          });
+          it('bee weights add to 1', () => {
+            expect(bees.reduce((prev, curr) => prev + curr.weight, 0)).toEqual(1);
```

Comment:
```suggestion
            expect(bees.reduce((prev, curr) => prev + curr.weight, 0)).toBeCloseTo(1);
```

Since we're doing floating-point math, we probably want to use Jasmine's `toBeCloseTo` matcher (otherwise this test could fail because `0.99999999974 !== 1` or something like that :stuck_out_tongue_winking_eye:)

(By default, `toBeCloseTo` is accurate to two decimal places, but you can change the precision if you want!)

Reply:
```suggestion
            expect(bees.reduce((prev, curr) => prev + curr.weight, 0)).toBeCloseTo(1, 10);
```
What do you think about expecting at least 10 decimal places?